shadowed-bung-sdk/pom.xml

```diff
@@ -0,0 +1,200 @@
+                            </relocations>
+                            <dependencyReducedPomLocation>target/reduced-pom.xml</dependencyReducedPomLocation>
+                        </configuration>
+                    </execution>
+                </executions>
+                <dependencies>
+                    <!--                    <dependency>-->
+                    <!--                        <groupId>com.github.bunq</groupId>-->
+                    <!--                        <artifactId>sdk_java</artifactId>-->
+                    <!--                        <version>${bunq-sdk.version}</version>-->
+                    <!--&lt;!&ndash;                        <exclusions>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                            <exclusion>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                                <groupId>com.squareup.okio</groupId>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                                <artifactId>okio</artifactId>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                            </exclusion>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                            <exclusion>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                                <groupId>com.squareup.okhttp3</groupId>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                                <artifactId>okhttp</artifactId>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                            </exclusion>&ndash;&gt;-->
+                    <!--&lt;!&ndash;                        </exclusions>&ndash;&gt;-->
+                    <!--                    </dependency>-->
+                </dependencies>
+            </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>flatten-maven-plugin</artifactId>
+                <version>${flatten-maven-plugin.version}</version>
+                <configuration>
+                    <outputDirectory>${project.basedir}/target</outputDirectory>
+                    <flattenMode>clean</flattenMode>
+                    <pomElements>
+                        <name/>
+                    </pomElements>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>flatten</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>flatten</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+
+    <repositories>
+        <repository>
+            <id>maven-central</id>
+            <url>https://repo.maven.apache.org/maven2</url>
+        </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
+
+</project>
```

Comment:
Clean up file  from comments and unused props 🧹
Add comment about unfamiliar `<optional>true</optional>` property

Reply:
fixed